### PR TITLE
[CodeCompletion] Fix non-determinisc failures in dependency check test cases

### PR DESCRIPTION
**Pull request by rintaro**
Sleep a few seconds to ensure the timestamps of the modified files are different from the last completion.

rdar://problem/62923248

### CODE REVIEW

**Comment by rintaro** [comment]
@swift-ci Please test

File: test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift [hunk 1]
```diff
@@ -36,6 +35,7 @@ func foo() {
 // RUN:   -req=complete -pos=5:3 %s -- ${COMPILER_ARGS[@]} == \
 
 // RUN:   -shell -- echo '### Modify bridging header library file' == \
+// RUN:   -shell -- sleep $SLEEP_TIME == \
 // RUN:   -shell -- cp -R $INPUT_DIR/MyProject_mod/LocalCFunc.h %t/MyProject/ == \
 // RUN:   -shell -- sleep $SLEEP_TIME == \
```
Comment (akyrtzi) [inline test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift:40]: Is the second sleep really needed?

Comment (rintaro) [reply test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift:40]: I believe it's not needed, but just I wanted to be on the safe side.

**Comment by rintaro** [comment]
@swift-ci Please test

File: test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift (hunk 1 above)
Comment (akyrtzi) [reply test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift:40]: Could we try removing it, it's better if we don't make tests unnecessarily slower.

**Comment by rintaro** [comment]
@swift-ci Please test

**Comment by rintaro** [comment]
@swift-ci Please test Linux

**Review by akyrtzi — APPROVED** [review]

**Comment by rintaro** [comment]
@swift-ci Please smoke test Linux

**Comment by rintaro** [comment]
@swift-ci Please smoke test Linux

**Pull request merged**